src/camera_session.cpp

```diff
@@ -94,7 +98,15 @@ int CameraSession::start() {
   }
 
   camera_->requestCompleted.connect(this, &CameraSession::requestComplete);
-  sink_ = std::make_unique<KMSSink>("");
+
+#ifdef HAVE_SDL
```

Comment:
A #else is different to an else, I know this looks funny but you want something like this here:

```
#ifdef HAVE_SDL
  if (opts.opt_sdl)
    sink_ = std::make_unique<SDLSink>();
  else
#endif
  {
    sink_ = std::make_unique<KMSSink>("");
  }
```

so it works for all cases